velox/exec/tests/DriverTest.cpp

```diff
@@ -961,6 +961,41 @@ class ThrowNodeFactory : public Operator::PlanNodeTranslator {
   uint32_t driversCreated{0};
 };
 
+class BlockedNoFutureNode : public ThrowNode {
+ public:
+  BlockedNoFutureNode(
+      const core::PlanNodeId& id,
+      OperatorMethod throwingMethod,
```

Comment:
Why does this node require throwingMethod? Does BlockedNoFutureNode need to use ThrowNode as the base class?

Reply:
Fixed by making `BlockedNoFutureNode`, `BlockedNoFutureOperator`, and `BlockedNoFutureNodeFactory` extends the `Planode`, `Operator`, and `PlanNodeTranslator` directly.